cypress/integration/recieve-notice/recieve-notice-after-checkout.spec.js

```diff
@@ -0,0 +1,133 @@
+            console.log(points);
+          });
+        });
+      })
+      .then(() => {
+        cy.getMaterialTypes({ limit: 1 }).then((res) => { materialType = res.id; });
+        cy.getLocations({ limit: 1 }).then((res) => { location = res.id; });
+        cy.getHoldingTypes({ limit: 1 }).then((res) => { holdingType = res[0].id; });
+        cy.getHoldingSources({ limit: 1 }).then((res) => { holdingSource = res[0].id; });
+        cy.getInstanceTypes({ limit: 1 }).then((res) => { instanceType = res[0].id; });
+        cy.getLoanTypes({ limit: 1 }).then((res) => { loanType = res[0].id; });
+      })
+      .then(() => {
+        cy.createInstance({
+          instance: {
+            instanceTypeId: instanceType,
+            title: `Pre-checkout instance ${Number(new Date())}`,
+          },
+          holdings: [{
+            holdingsTypeId: holdingType,
+            permanentLocationId: location,
+            sourceId: holdingSource,
+          }],
+          items: [
+            [{
+              barcode: ITEM_BARCODE,
+              missingPieces: '3',
+              numberOfMissingPieces: '3',
+              status: { name: 'Available' },
+              permanentLoanType: { id: loanType },
+              materialType: { id: materialType },
+            }],
+          ],
+        });
+      })
+      .then(() => {
+        noticePolicyTemplate.createViaApi(TEMPLATE_CATEGORIES.loan).then(res => { templateId = res.body.id; });
+      })
+      .then(() => {
+        noticePolicy.createWithTemplateApi(templateId, NOTICE_ACTIONS.checkout).then(res => { noticePolicyId = res.id; });
+      })
+      .then(() => {
+        cy.loginAsAdmin({ path: topMenu.checkOutPath, waiter: checkout.waitLoading });
+      })
+      .then(() => {
+        cy.checkOutItem(userData.username, ITEM_BARCODE);
+      });
+  });
+  afterEach('Deleting created entities', () => {
+    circulationRules.deleteAddedRuleApi(Cypress.env('defaultRules'));
+    noticePolicy.deleteApi(noticePolicyId);
+    noticePolicyTemplate.deleteViaApi(templateId);
+    checkInActions.createItemCheckinApi({
+      itemBarcode: ITEM_BARCODE,
+      servicePointId: userServicePoint,
+      checkInDate: moment.utc().format(),
+    }).then(() => {
+      cy.deleteUser(userData.id);
+      patronGroups.deleteViaApi(patronGroup.id);
+    });
+    cy.getInstance({ limit: 1, expandAll: true, query: `"items.barcode"=="${ITEM_BARCODE}"` })
+      .then((instance) => {
+        cy.deleteItem(instance.items[0].id);
+        cy.deleteHoldingRecord(instance.holdings[0].id);
+        cy.deleteInstanceApi(instance.id);
+      });
+  });
+
+  it('C347621 Check that user can receive notice with multiple items after finishing the session "Check out" by clicking the End Session button', { tags: [testTypes.smoke, devTeams.firebird] }, () => {
+    cy.visit(topMenu.circulationLogPath);
+    searchPane.searchByItemBarcode(ITEM_BARCODE);
+    searchPane.verifyResultCells();
```

Comment:
can't see verification that filtered out row is expected by presented values

Reply:
added lower + changed creating template and notice from API to GUI